src/sumOfOther.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function sumOfOther(arr) {
+  const sum = arr.reduce((acc, curr) => acc + curr);
+  const arrOfSums = arr.map(array => sum - array);
```

Comment:
Please, notice that we have to choose names of variables due to their sense. In this case there should be smth like "number", or at least "element", but not "array".